pkg/linode-bs/deploy/kubernetes/08-cm-get-linode-id.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: get-linode-id
+  namespace: kube-system
+  labels:
+    app: csi-linode
+data:
+  get-linode-id.sh: |-
+    #!/bin/bash -efu
+    id="$(kubectl get node/"${NODE_NAME}" -o jsonpath='{.metadata.annotations.lke\.linode\.com/linode-id}')"
```

Comment:
Once the Linode CCM is fully initialized, `Node.spec.providerID` will also contain the Linode ID in the format: `linode://<LinodeID>`. This should look for that as well, as a fallback.

Reply:
Realizing that the provider ID exists and how it is populated in the node object makes me believe that the `linode-id` annotation is probably at best redundant. I'll keep that in mind for future clean-up.